In-house patch to change how angularjs is imported.  In
XStatic-Angular-1.3.7.0, angular_cookies and angular_mock were rolled
into the angular pacakge.

--- horizon-2014.2.1/openstack_dashboard/settings.py.orig	2015-01-15 19:27:56.738322606 -0700
+++ horizon-2014.2.1/openstack_dashboard/settings.py	2015-01-15 19:28:26.944716700 -0700
@@ -24,8 +24,6 @@ import warnings
 from django.utils.translation import ugettext_lazy as _
 import xstatic.main
 import xstatic.pkg.angular
-import xstatic.pkg.angular_cookies
-import xstatic.pkg.angular_mock
 import xstatic.pkg.bootstrap_datepicker
 import xstatic.pkg.bootstrap_scss
 import xstatic.pkg.d3
@@ -155,10 +153,6 @@ STATICFILES_FINDERS = (
 STATICFILES_DIRS = [
     ('horizon/lib/angular',
         xstatic.main.XStatic(xstatic.pkg.angular).base_dir),
-    ('horizon/lib/angular',
-        xstatic.main.XStatic(xstatic.pkg.angular_cookies).base_dir),
-    ('horizon/lib/angular',
-        xstatic.main.XStatic(xstatic.pkg.angular_mock).base_dir),
     ('horizon/lib/bootstrap_datepicker',
         xstatic.main.XStatic(xstatic.pkg.bootstrap_datepicker).base_dir),
     ('bootstrap',
